react-apps/self-care-portal/self-care-portal-frontend/accelerator/src/landing_page/Home.jsx

```diff
@@ -28,6 +28,7 @@ import { UserContext } from "../context/UserContext";
 import { ConsentContext } from "../context/ConsentContext";
 import { AppInfoContext } from "../context/AppInfoContext";
 import { HomeTile } from "./HomeTile.jsx";
+import { logout } from "../login/logout.js";
```

Comment:
What is the usage of `logout` here? If there are no changes in this file, shall we keep the original one?

Reply:
fixed in e3f02ac73e2598e97bbef38524ad149927667e94